pkg/apis/sources/v1alpha1/mongodbSource_types_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+func TestGroupVersionKind(t *testing.T) {
+	src := CouchDbSource{}
```

Comment:
MongoDbSource?

Reply:
same in other places

Reply:
Yup, I hadnt touched the test files, I shouldnt have uploaded them